### ISSUE TITLE
Add C compillers

### ISSUE THREAD

**Issue by DanielMazurkiewicz** [kozross/awesome-c#153]
https://bellard.org/tcc/
https://github.com/rui314/8cc
https://github.com/drh/lcc

**Comment by kozross** [comment]
Does TCC have a new maintainer? Last I checked, the page still lists it as not being worked on.

I am not sure on the maintenance status of `8cc` and `lcc` - they both look (at least semi) abandoned.

**Comment by cjwagenius** [comment]
Yes. There are some people still working on it. It's pretty active.

git://repo.or.cz/tinycc.git
